css/loadingSpinner.scss

```diff
@@ -0,0 +1,40 @@
+$offset: 187;
+$duration: 1.4s;
+
+.yoast-svg-icon-loading-spinner {
+	animation: rotator $duration linear infinite;
+}
+
+@keyframes rotator {
+	0% { transform: rotate( 0deg ); }
+	100% { transform: rotate( 270deg ); }
+}
+
+.path {
+	stroke-dasharray: $offset;
+	stroke-dashoffset: 0;
+	transform-origin: center;
+	animation:
+		dash $duration ease-in-out infinite,
+		colors ( $duration * 4 ) ease-in-out infinite;
+}
+
+@keyframes colors {
+	0% { stroke: #64a60a; }
+	25% { stroke: #64a60a; }
+	50% { stroke: #64a60a; }
+	75% { stroke: #64a60a; }
+	100% { stroke: #64a60a; }
+}
```

Comment:
Why it is using this animation for the color even if the color is always the same? As far as I see, this could be removed and the color set on the `.path` selector with `stroke: #64a60a;`?